### PR TITLE
Store Test Results In JSON

### LINKED ISSUE #1493: Store Test Results In JSON
This would allow test runners to automatically save their data, instead of requiring someone to manually input it.

### PR DESCRIPTION
**Pull request by TheBrokenRail**
Fixes #1493

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -548,7 +560,7 @@ function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
       .append($('<td></td>')
         .attr('id', getHtmlId(id))
         .append('<span><a class="anchor" href="#' + getHtmlId(id) + '">&sect;</a>' + name + footnoteHTML(t) + '</span></td>')
-        .append(testScript(t.exec, compiler, rowNum++))
+        .append(testScript(eval(t.exec), compiler, rowNum++))
```
Comment (ljharb) [inline build.js]: we reallllly shouldn't use `eval`. why is this needed?

**Review by ljharb — CHANGES_REQUESTED** [review]
The tests themselves should be in .js, since they're code. Only the *data* should (maybe) be in json

File: build.js (hunk 1 above)
Comment (TheBrokenRail) [reply build.js]: Because ```testScript``` expects a function.

Comment (ljharb) [reply build.js]: That’s why the function should be stored in a .js file - functions aren’t data, and json is for data.

Comment (TheBrokenRail) [reply build.js]: It now stores the tests in JS and the results in JSON.

**Comment by zloirock** [comment]
Now:
image
This PR:
image

**Comment by zloirock** [comment]
@TheBrokenRail any progress?
